app/lib/getOpeningTimesMessage.js

```diff
@@ -0,0 +1,38 @@
+function getOpeningHoursMessage(status) {
+  if (status.isOpen) {
+    const closedNext = status.until;
```

Comment:
See comments in other PR. Happy to discuss the merits of different names via slack tomorrow.